src/Sql/Tools/dbo/Tables/Send.sql

```diff
@@ -14,12 +14,13 @@
     [DeletionDate]   DATETIME2 (7)    NOT NULL,
     [Disabled]       BIT              NOT NULL,
     [HideEmail]      BIT              NULL,
+    [CipherId]       UNIQUEIDENTIFIER NULL
     CONSTRAINT [PK_Send] PRIMARY KEY CLUSTERED ([Id] ASC),
     CONSTRAINT [FK_Send_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]),
     CONSTRAINT [FK_Send_User] FOREIGN KEY ([UserId]) REFERENCES [dbo].[User] ([Id])
 );
```

Comment:
:x: This is missing a foreign key relationship to the Cipher-Table

Reply:
Foreign Key added